src/bun.js/bindings/webcore/JSURLSearchParams.cpp

```diff
@@ -459,7 +460,8 @@ JSC::JSValue getInternalProperties(JSC::VM& vm, JSC::JSGlobalObject* lexicalGlob
 
                 array->initializeIndex(initializationScope, 0, jsValue);
                 array->initializeIndex(initializationScope, 1, stringResult);
-                obj->putDirect(vm, ident, array, 0);
+                obj->putDirectMayBeIndex(lexicalGlobalObject, ident, array);
+                RETURN_IF_EXCEPTION(throwScope, {});
```

Comment:
Is this `RETURN_IF_EXCEPTION` necessary?

Reply:
To appease the exception scope verifier yeah